clients/admin-ui/src/features/privacy-notices/cells.tsx

```diff
@@ -10,6 +11,29 @@ export const MechanismCell = (
   cellProps: CellProps<typeof MECHANISM_MAP, string>
 ) => <MapCell map={MECHANISM_MAP} {...cellProps} />;
 
+type TagNames = "available" | "enabled" | "inactive";
+
+const systemsApplicableTags: Record<TagNames, TagProps & { tooltip: string }> =
```

Comment:
I haven't seen this use of `Record` before. Nice!

Reply:
ah yeah, it's a nice way to declare a dictionary, esp if you already know what the keys will be!